packages/common/src/items/convert-initiative-to-template.ts

```diff
@@ -0,0 +1,80 @@
+import {
```

Comment:
I think this one should move into the hub-initiatives package... but we can do in next PR